requirements-test.txt

```diff
@@ -1,4 +1,4 @@
 pytest==6.2.5
 piexif==1.1.3
 Pillow>=9.5.0; implementation_name != "pypy" or python_version != "3.7"
-Pillow==9.3.0; implementation_name == "pypy" and python_version == "3.7"
+Pillow==10.3.0; implementation_name == "pypy" and python_version == "3.7"
```

Comment:
Pillow 9.4+ doesn't have precompiled binaries for python 3.7, this is the reason why 9.3 version was here

Reply:
It could worth remove py37 at all